definitions/infra-awsrekognition/dashboard.json

```diff
@@ -0,0 +1,125 @@
+{
+  "name": "AWS Rekognition",
+  "description": null,
+  "pages": [
+    {
+      "name": "AWS Rekognition",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Request Counts",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 8,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT count(aws.rekognition.CallCount) as `Total Requests`, count(aws.rekognition.SuccessfulRequestCount) as `Successful Requests`, count(aws.rekognition.ThrottledCount) as `Throttled Count`"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Errors Count",
```

Comment:
For consistency, the title should be "Errors count"

Reply:
addressed